AUTHORS

```diff
@@ -24,7 +24,7 @@ andresvia <andres.via@gmail.com>
 Andrew Dunham (andrew-d) <andrew@du.nham.ca>
 Andrew Rabert (nvllsvm) <ar@nullsum.net> <6550543+nvllsvm@users.noreply.github.com>
 Andrey D (scienmind) <scintertech@cryptolab.net> <scienmind@users.noreply.github.com>
-André Colomb <github.com@andre.colomb.de>
+André Colomb (acolomb) <src@andre.colomb.de>
```

Comment:
```suggestion
André Colomb (acolomb) <github.com@andre.colomb.de> <src@andre.colomb.de>
```

Reply:
GitHub used your GitHub address for the squashed commit, rather, so we need at minimum both in the file.

Reply:
Okay, fine with me.  Not really sensible for GitHub to do that, though.

Reply:
I agree.